llvm/tools/llvm-cas-object-format/llvm-cas-object-format.cpp

```diff
@@ -767,6 +768,23 @@ static Error createSplitEHFramePasses(jitlink::LinkGraph &G) {
   return Error::success();
 }
 
+static Error createSplitDebugLinePassess(jitlink::LinkGraph &G) {
```

Comment:
This is a pass, rather than a pass-creator, so `splitDebugLine` seems like a better name.